src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -215,14 +218,15 @@ const usePushNotificationHook = () => {
               createdAt: remoteMessage.data?.created_at,
               updatedAt: remoteMessage.data?.created_at,
               rawJson: {},
-              attachmentJson: remoteMessage.data?.attachment,
+              attachmentJson: remoteMessage.data?.attachments,
               user: null,
               status: remoteMessage.data?.status,
               isMe: false,
               isContinuous: false
             });
             await chatSchema.save(localDb);
           }
+          setIsLoadingFetchAllMessage(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 204, the change from `remoteMessage.data?.attachment` to `remoteMessage.data?.attachments` needs to be reflected across the entire codebase. Also, the addition of `setIsLoadingFetchAllMessage(false)` is good as it ensures the loading state is reset after the fetch operation. However, make sure to add error handling to reset the loading state in case of any errors during fetch operation.

```diff
- attachmentJson: remoteMessage.data?.attachment,
+ attachmentJson: remoteMessage.data?.attachments,
+ setIsLoadingFetchAllMessage(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->